packages/node/src/types.ts

```diff
@@ -20,7 +20,19 @@ export interface NodeOptions extends Options {
   /** HTTPS proxy certificates path */
   caCerts?: string;
 
-  /** Sets the number of context lines for each frame when loading a file. */
+  /**
+   * Sets the number of context lines for each frame when loading a file.
+   *
+   * @deprecated Context lines configuration has moved to the `ContextLines` integration, and can be used like this:
```

Comment:
Nice 👍. We'll need to update docs for this as well.

Reply:
Docs changes: https://github.com/getsentry/sentry-docs/pull/4746